src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -297,7 +305,8 @@
     <div>
         {{- form_label(form) -}}
         {{- form_errors(form) -}}
-        {{- form_widget(form) -}}
+        {{- form_widget(form, { 'helpBlockDisplayed': true }) -}}
```

Comment:
Couldn't we check if `help is not empty` here? 
That would make the logic in `widget_attributes` simpler

Reply:
The problem is when a user just display form_widget for exemple, the form_help is not call, then we don't have to display the aria-describedBy, because the block doesn't exists.

Reply:
I see. Could we try something like: 

`{{- form_widget(form, { 'helpBlockDisplayed': (help is not empty) }) -}}` or that may be a syntax error.

Reply:
It's the same problem, we can't do it automatically, the help may be set, but don't displayed if the user do:
    
    {{ form_label(form.name) }}
    {{ form_errors(form.name) }}
    {{ form_widget(form.name) }}

The, he doesn't call `form_label`, then label is not displayed, but set in the form.

Reply:
Yeah, But it that a problem? 

If the user do: 
```
{{ form_label(form.name) }}
{{ form_errors(form.name) }}
{{ form_widget(form.name) }}
```
Then **no** form help will be rendered and no `aria-describedBy` will be used. 

--------

If the user do: 
```
{{ form_label(form.name) }}
{{ form_errors(form.name) }}
{{ form_widget(form.name) }}
{{ form_help(form.name) }}
```
Then form help **will** be rendered but no `aria-describedBy` will be used. 

------

We should only make sure that the logic is clean and when the user do:

```
{{ form_row(form.name) }}
```

Then form help **will** be rendered AND `aria-describedBy` will be used.

Reply:
Yes, we can do that, I've do it to permit the user to add the aria-describedBy when don't use `form_row`.